contrib/subrepo-cheri-c-tests/test_runtime.c

```diff
@@ -386,7 +400,7 @@ capsighandler(int signo, siginfo_t *info __unused, ucontext_t *uap)
 	}
```

Comment:
I feel like this fprintf should always be enabled.

Reply:
That's an old bug, but basically it seems to be a manual assert() invocation.